### ISSUE TITLE
How to set the size of the GeoJSON display output?

### ISSUE THREAD

**Issue by gcbgit** [jupyter/help#338]
I'm a little foggy on the line between IPython and Jupyter, so I'll start by asking here.  I've recently found the IPython.display.GeoJSON, which gives a lot of capability for so little user-side code.

The problem I'm having is that I don't seem able to set the size of the map display window as I could with the width and height parameters of an image, or the figsize of a matlab plot.  All of my maps are browser width wide, but not very tall.

Is there a way to set the size?

**Comment by SylvainCorlay** [comment]
@gcbgit I don't think that `IPython.display.GeoJSON` can do this, since it is a very basic default repr for geojson data.

I would recommend you to check out the https://github.com/jupyter-widgets/ipyleaflet project which allows to interactively modify maps, change the basemap, display geojson data on the top of it, etc.
